manual/Website1/src/App.jsx

```diff
@@ -1,18 +1,28 @@
 import React, { Component } from 'react';
-import { ExternalComponent,corsImport } from 'webpack-external-import';
+import {
+  ExternalComponent, corsImport, getChunkPath, getChunkDependencies, importDependenciesOf,
+} from 'webpack-external-import';
 import HelloWorld from './components/goodbye-world';
 
 import('moment');
 class App extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      titleUrl: null,
+      manifestLoaded: false,
+      loaded: false,
+    };
   }
 
   componentDidMount() {
     corsImport('http://localhost:3002/importManifest.js').then(() => {
       this.setState({ manifestLoaded: true });
-      import(/* webpackIgnore:true */`http://localhost:3002/${window.entryManifest['website-two']['SomeExternalModule.js']}`).then(() => {
+      importDependenciesOf('http://localhost:3002', 'website-two', 'TitleComponent').then((url) => {
+        this.setState({ titleUrl: url });
+      });
+
+      import(/* webpackIgnore:true */getChunkPath('http://localhost:3002', 'website-two', 'SomeExternalModule.js')).then(() => {
```

Comment:
How do we weave import into importDependencyOf without falling back to something crappy like load.js, using native browser imports with polyfills would be cool to keep